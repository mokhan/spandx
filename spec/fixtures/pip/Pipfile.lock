{
    "_meta": {
        "hash": {
            "sha256": "d9b5cc506fc4feb9bf1ae7cadfd3737d5a0bd2b2d6c3fbcf0de3458bab34ad89"
        },
        "pipfile-spec": 6,
        "requires": {
            "python_version": "3.8"
        },
        "sources": [
            {
                "name": "pypi",
                "url": "https://pypi.org/simple",
                "verify_ssl": true
            }
        ]
    },
    "default": {
        "six": {
            "hashes": [
                "sha256:1f1b7d42e254082a9db6279deae68afb421ceba6158efa6131de7b3003ee93fd",
                "sha256:30f610279e8b2578cab6db20741130331735c781b56053c59c4076da27f06b66"
            ],
            "index": "pypi",
            "version": "==1.13.0"
        }
    },
    "develop": {}
}
