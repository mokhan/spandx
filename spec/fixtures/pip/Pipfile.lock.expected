+------+---------+----------+--------------------------------+
| Name | Version | Licenses | Location                       |
+------+---------+----------+--------------------------------+
| six  | 1.13.0  | MIT      | spec/fixtures/pip/Pipfile.lock |
+------+---------+----------+--------------------------------+
